config/config-server/resources/cruise-config.xsd

```diff
@@ -143,19 +143,52 @@
     <xsd:complexType name="configRepoType">
         <xsd:sequence>
             <xsd:choice minOccurs="1" maxOccurs="1">
-                <xsd:element name="svn" type="svnType"/>
-                <xsd:element name="hg" type="hgType"/>
-                <xsd:element name="p4" type="p4Type"/>
-                <xsd:element name="git" type="gitType"/>
-                <xsd:element name="tfs" type="tfsType"/>
-                <xsd:element name="scm" type="scmReferenceType"/>
```

Comment:
why is `scmReferenceType` removed, it should be supported through the API as well.